data-types.md

```diff
@@ -238,7 +251,35 @@ Examples of mode-specific `trip_attributes`:
 
 [Top][toc]
 
-### Reservation Type
+### Fares
```

Comment:
I think Fares have been moved into each mode's `fare_attributes` field within the [trip object](https://github.com/openmobilityfoundation/mobility-data-specification/tree/dev/provider#trips), instead of a stand alone object for all modes.

https://github.com/openmobilityfoundation/mobility-data-specification/blob/dev/modes/passenger-services.md#fare-attributes

Reply:
And oops, it looks like we need a `fare_attributes` field in the trips object, where each mode can put it's attributes.